cpp/velox/substrait/SubstraitToVeloxPlan.cc

```diff
@@ -1449,10 +1450,14 @@ connector::hive::SubfieldFilters SubstraitToVeloxPlanConverter::createSubfieldFi
       auto expr = scalarFunction.arguments()[0].value();
       if (expr.has_scalar_function()) {
         // Set its child to filter info with reverse enabled.
-        setFilterInfo(scalarFunction.arguments()[0].value().scalar_function(), inputTypeList, columnToFilterInfo, true);
+        setFilterInfo(expr.scalar_function(), inputTypeList, columnToFilterInfo, true);
+      } else if (expr.has_singular_or_list()) {
+        // Since currently only integer and string types in "IN" expression
+        // can be pushed down, Options types should be checked
```

Comment:
> Options types should be checked

Could you remind me where the check for types is added?

Reply:
Sorry, the comment is useless